CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # CHANGELOG
 
+## 1.2.1 - 2017-08-18
```

Comment:
Can you just make this 1.2.0? 1.2.0 never shipped, so 1.2.1 doesn't make sense here.